emails/tracker_lists/level-one-tracker.json

```diff
@@ -1 +1 @@
com", "open.mkt685.com", "open.mkt6882.com", "open.mkt6903.com", "open.mkt6917.com", "open.mkt6967.com", "open.mkt71.net", "open.mkt7234.com", "open.mkt7580.com", "open.mkt7596.com", "open.mkt7752.com", "open.mkt7783.com", "open.mkt7832.com", "open.mkt7842.com", "open.mkt7883.com", "open.mkt7946.com", "open.mkt7971.com", "open.mkt7972.com", "open.mkt7974.com", "open.mkt8007.com", "open.mkt8008.com", "open.mkt8043.com", "open.mkt8062.com", "open.mkt8063.com", "open.mkt8064.com", "open.mkt8096.com", "open.mkt81.net", "open.mkt8133.com", "open.mkt8163.com", "open.mkt8267.com", "open.mkt829.com", "open.mkt8345.com", "open.mkt8369.com", "open.mkt8586.com", "open.mkt8628.com", "open.mkt8756.com", "open.mkt8763.com", "open.mkt8988.com", "open.mkt9026.com", "open.mkt9054.com", "open.mkt912.com", "open.mkt9203.com", "open.mkt922.com", "open.mkt941.com", "open.mkt9430.com", "open.mkt9775.com", "open.mkt9862.com", "open.mkt9923.com", "open.mkt9942.com", "basiscommunicatie.nl", "acblnk.com", "acmbtrc.com", "acmtrk.com", "clickacumba.com", "emlmkt.com", "mailmktool.com", "showlanding.com", "trckacbm.com", "pippio.com", "adform.net", "2o7.net", "demdex.net", "omtrdc.net", "adroll.com", "adspeed.net", "adsugar.ch", "aforesponse.com", "agilemeasure.com", "authoremail.com", "apms5.com", "autopilotmail.io", "bandzoogle.com", "barilliance.com", "bemail.it", "bentonow.com", "cdn.uk.exponea.com", "sptracking.getblueshift.com", "thebrighttag.com", "cp20.com", "cpro20.com", "cpro30.com", "skem1.com", "api.carrotquest.io", "cdnwidget.com", "cl1-api.connectif.cloud", "eu2-api.connectif.cloud", "eu3-api.connectif.cloud", "eu4-api.connectif.cloud", "convertkit-mail.com", "convertkit-mail2.com", "convertkit-mail3.com", "convertkit-mail4.com", "convertkit-mail5.com", "convertkit-mail6.com", "e.customeriomail.com", "edrone.me", "emaillabs.co", "engagingnetworks.app", "ot.gliq.com", "track.gliq.com", "doubleclick.net", "google-analytics.com", "gasv1.com", "infernotions.com", "email-messaging.com", "ixactcontact.com", "mailer.lassocrm.com", "leadersend.com", "track-mb.bra2hmail.com", "liadm.com", "m3651.net", "m3652.net", "mailblue.eu", "eds5.mailcamp.nl", "agentofficemail.com", "mandrillapp.com", "mailcamp.net.pl", "mailtracker.pl", "senderit.pl", "xylionmail.pl", "mathtag.com", "api-01.moengage.com", "agkn.com", "myvisualiq.net", "nyl.as", "openeducat.org", "pvd.to", "pstmrk.it", "pushnami.com", "opens.responder.co.il", "system.send24.pl", "tracking.retailrocket.net", "sailplay.ru", "krxd.net", "pixel.inbox.exacttarget.com", "ondemand.com", "segment.io", "sendcloud.net", "sendfox.com", "sendiio.app", "sendpul.se", "sitescout.com", "smlists.com", "smartsendy.com", "smtp2go.com", "smtp2go.net", "servedbyadbutler.com", "strikestack.com", "hodes.com", "tealiumiq.com", "adventure-novels.com", "returnpath.net", "vinc.fr", "viralhosts.com", "trac.visme.co", "stable.cz", "campaign-tracking.woowup.com", "e.wordfly.com", "xpressmail.hu", "xtremepush.com", "em.yotpo.com", "p.yotpo.com"]
```

Comment:
Can you run this file through a JSON pretty-printer like `jq`?

Reply:
cbdab0c